apis/base.es6.js

```diff
@@ -25,17 +25,6 @@ class BaseAPI {
     this.config = base.config;
     this.cache = base.cache;
     this.event = base.event;
-
-    if (base.config) {
-      this.origin = base.config.origin;
-
-      if (base.config.origins) {
```

Comment:
This was here originally so that you could, say, include an `ads` api with a separate origin. This probably needs to be refactored, as it won't really work as-is.